virttest/utils_misc.py

```diff
@@ -3470,6 +3470,30 @@ def check_device_driver(pci_id, driver_type):
     return driver == driver_type
 
 
+def get_guest_os_datetime(vm, os_type):
```

Comment:
function name is good, but return type looks not so good.
I think if output is invalid please raise test.Error not return False directly. 
And please follow avocado doc string style to add your doc-string.